google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/gcs/GcsBucketContentEditorPanelTest.java

```diff
@@ -88,6 +88,7 @@ public void testEmptyBucket_noBucketContentTable() {
     JTable bucketTable = editorPanel.getBucketContentTable();
     assertThat(bucketTable.getColumnCount()).isEqualTo(0);
     assertThat(bucketTable.getRowCount()).isEqualTo(0);
+    assertFalse(editorPanel.getLoadingPanel().isVisible());
```

Comment:
Is there another assertion you can add elsewhere to verify the panel is shown before the call is done? And what happens when the call fails?